core/templates/dev/head/domain/exploration/ReadOnlyExplorationBackendApiService.js

```diff
@@ -75,16 +85,19 @@ oppia.factory('ReadOnlyExplorationBackendApiService', [
        * retrieves the exploration from the backend, it will store it in the
        * cache to avoid requests from the backend in further function calls.
        */
-      loadExploration: function(explorationId) {
+      loadExploration: function(explorationId, version) {
         return $q(function(resolve, reject) {
-          if (_isCached(explorationId)) {
+          if (_isCached(explorationId) && !version) {
             if (resolve) {
               resolve(angular.copy(_explorationCache[explorationId]));
             }
           } else {
-            _fetchExploration(explorationId, function(exploration) {
+            _fetchExploration(explorationId, version, function(exploration) {
               // Save the fetched exploration to avoid future fetches.
-              _explorationCache[explorationId] = exploration;
+              // Excludes version for now.
```

Comment:
Please mention this in the documentation for ``loadExploration`` so other developers are aware of this constraint. Should we instead have a ``loadExploration`` and ``loadLatestExploration``? It might be clearer than having an optional version parameter.